# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookieService.cpp
# Commit: d2381f88168a
# Full Hash: d2381f88168ae15ad51e03a193c81f0267b95b7c
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2024-12-17 17:00:49
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 6: Add tests. r=cookie-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223386
# ==============================================================================

diff -r 675785853d7a -r d2381f88168a netwerk/cookie/CookieService.cpp
--- a/netwerk/cookie/CookieService.cpp	Tue Dec 17 08:57:31 2024 +0000
+++ b/netwerk/cookie/CookieService.cpp	Tue Dec 17 08:57:31 2024 +0000
@@ -1814,5 +1814,14 @@
   return NS_OK;
 }
 
+NS_IMETHODIMP
+CookieService::TestGet3PCBExceptions(nsTArray<nsCString>& aExceptions) {
+  aExceptions.Clear();
+
+  mThirdPartyCookieBlockingExceptions.GetExceptions(aExceptions);
+
+  return NS_OK;
+}
+
 }  // namespace net
 }  // namespace mozilla